src/coreclr/debug/shared/dbgtransportsession.cpp

```diff
@@ -156,7 +156,9 @@ HRESULT DbgTransportSession::Init(DebuggerIPCControlBlock *pDCB, AppDomainEnumer
         Release();
         return E_OUTOFMEMORY;
     }
-
+#if defined(__linux__) || defined(__FreeBSD__)
```

Comment:
Why call this in Linux and FreeBSD only? We do not have this ifdef around other places that call `SetThreadDescription`.

Reply:
It is to figure out why CI fails on Windows tests.
With this `ifdef` I was able to see the code actually affects the CI results.

https://github.com/dotnet/runtime/blob/af1e6ba6414d5794d751868c9762dad2e95e8fc5/src/coreclr/utilcode/winfix.cpp#L236
`SetThreadDescription` is wrapped to use like the above in Windows and `SetThreadName` is provided.
I just changed the code to use it and waiting for CI result.
Hope this will pass. :)

Reply:
ok, please let us know once you think that the PR is ready